src/iplant_groups/routes/schemas/privileges.clj

```diff
@@ -5,9 +5,23 @@
             [iplant_groups.routes.schemas.folder :as folder]
             [schema.core :as s]))
 
-(def ValidFolderPrivileges (s/enum "create" "stem" "stemAdmin" "stemAttrRead" "stemAttrUpdate"))
```

Comment:
Grouper was complaining about `stemAdmin` when I was using it for testing, so I removed it from the set of options.